x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/140_metadata.yml

```diff
@@ -5,7 +5,7 @@ setup:
         - method: POST
           path: /_query
           parameters: [method, path, parameters, capabilities]
-          capabilities: [metadata_fields, metadata_field_ignored]
+          capabilities: [metadata_fields, metadata_ignored_field]
```

Comment:
This is unrelated, but it turns out this test hadn't been running at all due to an incorrect capability name.